pkg/manifest/build.go

```diff
@@ -45,6 +45,7 @@ func (p *Build) addDependent(dep Pipeline) {
 func (p *Build) getPackageSetChain(distro Distro) []rpmmd.PackageSet {
 	// TODO: make the /usr/bin/cp dependency conditional
 	// TODO: make the /usr/bin/xz dependency conditional
+	// TODO: make the openscap-utils package conditional
```

Comment:
Is `openscap-utils` going to be included here?

Reply:
I had it here first in testing and changed it... I must have missed this. Will fix